[2024-09-25 23:57:29,567: INFO: main: Welcome]
[2024-09-26 00:45:28,370: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 00:47:22,049: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 00:50:11,180: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 00:50:38,808: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 01:30:28,173: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 01:30:28,204: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 01:39:57,491: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-26 01:39:57,494: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 01:39:57,498: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 01:39:57,594: INFO: common: created directory at: artifacts]
[2024-09-26 01:39:57,609: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-26 01:39:57,610: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 01:40:19,641: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 01:40:21,658: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-26 01:40:21,658: INFO: main: *******************]
[2024-09-26 01:40:21,658: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-09-26 01:40:21,660: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 01:40:21,662: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 01:40:21,662: INFO: common: created directory at: artifacts]
[2024-09-26 01:40:21,663: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 01:40:25,987: ERROR: main: `load_weights` requires h5py package when loading weights from HDF5. Try installing h5py.]
Traceback (most recent call last):
  File "D:\kidney_disease_cnn_deeplearning\main.py", line 29, in <module>
    prepare_base_model.main()
  File "d:\kidney_disease_cnn_deeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 17, in main
    prepare_base_model.get_base_model()
  File "d:\kidney_disease_cnn_deeplearning\src\cnnClassifier\components\prepare_base_model.py", line 15, in get_base_model
    self.model = tf.keras.applications.vgg16.VGG16(
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\applications\vgg16.py", line 248, in VGG16
    model.load_weights(weights_path)
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\saving\legacy\save.py", line 471, in load_weights
    raise ImportError(
ImportError: `load_weights` requires h5py package when loading weights from HDF5. Try installing h5py.
[2024-09-26 01:42:32,281: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-26 01:42:32,284: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 01:42:32,286: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 01:42:32,414: INFO: common: created directory at: artifacts]
[2024-09-26 01:42:32,416: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-26 01:42:32,417: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 01:42:53,439: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 01:42:54,910: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-26 01:42:54,911: INFO: main: *******************]
[2024-09-26 01:42:54,912: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-09-26 01:42:54,914: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 01:42:54,915: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 01:42:54,915: INFO: common: created directory at: artifacts]
[2024-09-26 01:42:54,916: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 01:42:57,177: ERROR: main: `load_weights` requires h5py package when loading weights from HDF5. Try installing h5py.]
Traceback (most recent call last):
  File "D:\kidney_disease_cnn_deeplearning\main.py", line 29, in <module>
    prepare_base_model.main()
  File "d:\kidney_disease_cnn_deeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 17, in main
    prepare_base_model.get_base_model()
  File "d:\kidney_disease_cnn_deeplearning\src\cnnClassifier\components\prepare_base_model.py", line 15, in get_base_model
    self.model = tf.keras.applications.vgg16.VGG16(
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\applications\vgg16.py", line 248, in VGG16
    model.load_weights(weights_path)
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\saving\legacy\save.py", line 471, in load_weights
    raise ImportError(
ImportError: `load_weights` requires h5py package when loading weights from HDF5. Try installing h5py.
[2024-09-26 01:47:35,476: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 01:47:35,507: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 01:47:35,606: INFO: common: created directory at: artifacts]
[2024-09-26 01:47:35,618: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 02:04:19,324: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-26 02:04:19,341: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:04:19,369: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:04:19,447: INFO: common: created directory at: artifacts]
[2024-09-26 02:04:19,448: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-26 02:04:19,449: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 02:04:41,115: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 02:04:47,022: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-26 02:04:47,022: INFO: main: *******************]
[2024-09-26 02:04:47,022: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-09-26 02:04:47,025: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:04:47,027: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:04:47,027: INFO: common: created directory at: artifacts]
[2024-09-26 02:04:47,028: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 02:04:50,252: ERROR: main: `load_weights` requires h5py package when loading weights from HDF5. Try installing h5py.]
Traceback (most recent call last):
  File "D:\kidney_disease_cnn_deeplearning\main.py", line 29, in <module>
    prepare_base_model.main()
  File "d:\kidney_disease_cnn_deeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 17, in main
    prepare_base_model.get_base_model()
  File "d:\kidney_disease_cnn_deeplearning\src\cnnClassifier\components\prepare_base_model.py", line 15, in get_base_model
    self.model = tf.keras.applications.vgg16.VGG16(
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\applications\vgg16.py", line 248, in VGG16
    model.load_weights(weights_path)
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "D:\kidney_disease_cnn_deeplearning\venv\lib\site-packages\keras\saving\legacy\save.py", line 473, in load_weights
    raise ImportError(
ImportError: `load_weights` requires h5py package when loading weights from HDF5. Try installing h5py.
[2024-09-26 02:20:02,872: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:20:02,894: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:20:02,935: INFO: common: created directory at: artifacts]
[2024-09-26 02:20:02,937: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 02:34:35,971: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-26 02:34:36,106: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:34:36,142: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:34:36,143: INFO: common: created directory at: artifacts]
[2024-09-26 02:34:36,143: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-26 02:34:36,144: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 02:34:59,029: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 02:35:01,536: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-26 02:35:01,537: INFO: main: *******************]
[2024-09-26 02:35:01,537: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-09-26 02:35:01,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:35:01,541: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:35:01,542: INFO: common: created directory at: artifacts]
[2024-09-26 02:35:01,543: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 02:35:04,288: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-26 02:35:04,980: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-09-26 02:35:04,981: INFO: main: *******************]
[2024-09-26 02:35:04,982: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-09-26 02:35:04,984: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:35:04,986: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:35:04,987: INFO: common: created directory at: artifacts]
[2024-09-26 02:35:04,988: INFO: common: created directory at: artifacts\training]
[2024-09-26 02:35:54,108: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-09-26 02:35:54,109: INFO: main: *******************]
[2024-09-26 02:35:54,110: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-09-26 02:35:54,114: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:35:54,117: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:35:54,117: INFO: common: created directory at: artifacts]
[2024-09-26 02:36:12,350: INFO: common: json file saved at: scores.json]
[2024-09-26 02:36:12,352: INFO: common: json file saved at: scores.json]
[2024-09-26 02:36:12,352: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-09-26 02:40:42,522: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:40:42,529: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:40:42,531: INFO: common: created directory at: artifacts]
[2024-09-26 02:40:42,533: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 02:40:43,177: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-26 02:49:30,038: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 02:49:30,041: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 02:49:30,043: INFO: common: created directory at: artifacts]
[2024-09-26 02:49:30,045: INFO: common: created directory at: artifacts\training]
[2024-09-26 03:02:57,731: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-09-26 03:02:57,775: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 03:02:57,778: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 03:02:57,779: INFO: common: created directory at: artifacts]
[2024-09-26 03:02:57,779: INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-26 03:02:57,780: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 03:03:19,555: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-09-26 03:03:21,638: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-09-26 03:03:21,640: INFO: main: *******************]
[2024-09-26 03:03:21,640: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-09-26 03:03:21,643: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 03:03:21,644: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 03:03:21,645: INFO: common: created directory at: artifacts]
[2024-09-26 03:03:21,645: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-26 03:03:22,133: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-26 03:03:22,392: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-09-26 03:03:22,393: INFO: main: *******************]
[2024-09-26 03:03:22,393: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-09-26 03:03:22,396: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 03:03:22,398: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 03:03:22,398: INFO: common: created directory at: artifacts]
[2024-09-26 03:03:22,400: INFO: common: created directory at: artifacts\training]
[2024-09-26 03:13:21,322: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-09-26 03:13:21,325: INFO: main: *******************]
[2024-09-26 03:13:21,326: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-09-26 03:13:21,335: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 03:13:21,340: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 03:13:21,342: INFO: common: created directory at: artifacts]
[2024-09-26 03:13:47,019: INFO: common: json file saved at: scores.json]
[2024-09-26 03:13:47,021: INFO: common: json file saved at: scores.json]
[2024-09-26 03:13:47,022: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-09-26 03:19:26,115: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-09-26 03:19:26,118: INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-26 03:19:26,120: INFO: common: created directory at: artifacts]
[2024-09-26 03:19:45,836: INFO: common: json file saved at: scores.json]
[2024-09-26 03:19:48,777: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-09-26 03:19:49,559: INFO: builder_impl: Assets written to: C:\Users\AMITKU~1\AppData\Local\Temp\tmpo_j71354\model\data\model\assets]
